sql/src/test/scala/org/apache/sedona/sql/functions/FunctionsHelper.scala

```diff
@@ -0,0 +1,31 @@
+/*
+// * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@Imbruced You should use this comment style
/*
*
*
*/

Just like what you did in other file header